lib/src/components/NxFilterInput/NxFilterInput.scss

```diff
@@ -19,9 +20,25 @@
     }
   }
 
+  .nx-text-input__input::-webkit-calendar-picker-indicator {
+      display: none !important;
```

Comment:
If you use `visibility: hidden` you don't need the `!important`

Reply:
With `visibility: hidden`, it takes up space in the input field, but I will double check with design if it should be hidden.